x-pack/plugins/security/server/usage_collector/security_usage_collector.test.ts

```diff
@@ -35,6 +34,10 @@ describe('Security UsageCollector', () => {
   };
 
   const clusterClient = elasticsearchServiceMock.createLegacyClusterClient();
+  const collectorFetchClients = {
+    callCluster: clusterClient.asScoped().callAsCurrentUser,
+    esClient: elasticsearchServiceMock.createClusterClient().asInternalUser,
+  };
```

Comment:
I'd use `createCollectorFetchClientsMock` so we don't need to revisit this when we extend the options with more clients/context:
```suggestion
  const collectorFetchClients = createCollectorFetchClientsMock();
```